build-ps/percona-xtradb-cluster.spec

```diff
@@ -571,9 +621,13 @@ mkdir release
   # XXX: MYSQL_UNIX_ADDR should be in cmake/* but mysql_version is included before
   # XXX: install_layout so we can't just set it based on INSTALL_LAYOUT=RPM
   ${CMAKE} ../ -DBUILD_CONFIG=mysql_release -DINSTALL_LAYOUT=RPM \
+           -DDOWNLOAD_BOOST=1 -DWITH_BOOST=build-ps/boost \
```

Comment:
Just for record when we ported build-ps that developer uses for daily-run we observed few more options got added to PS-5.7 and we retained all of them. You may want to re-check this with PS-5.7-specs for more options.